source/includes/steps-enable-authentication-in-shardcluster-nodowntime-transition-config.yaml

```diff
@@ -59,15 +60,15 @@ pre: |
   down the primary and trigger an election. You can use the
   :method:`rs.status()` method to ensure the replica set elected a new primary.
   
-  Once you step down the primary, you can then shut it down using the
+  Once you step down the primary, you shut it down using the
```

Comment:
Because of the way they're organized, lines 58 - 61 seems to go with the code-block at line 54 while this paragraph would be post stepDown.
Instead, you might want to forestall someone accidentally shutting down the primary at lines 58-61 by prefacing with something like:

If upgrading the secondary, ...

If upgrading the primary,